packages/edit-site/src/components/sidebar/global-styles-sidebar.js

```diff
@@ -154,23 +161,19 @@ export default function GlobalStylesSidebar( {
 		getSetting,
 		setSetting,
 	} = useGlobalStylesContext();
-	const [ canRestart, onReset ] = useGlobalStylesReset();
 
-	if ( typeof blocks !== 'object' || ! root ) {
-		// No sidebar is shown.
```

Comment:
@youknowriad :wave: This was for preventing the GS sidebar from loading if the theme didn't have a `theme.json` (including if it has `experimental-theme.json`). As a result, we'll load the global sidebar for all themes, resulting in a weird situation: all panels are empty, clicking the color panel will propmts an error, clicking "reset to defaults" (for a theme with `experimental-theme.json` that was used in the past to store data) will also propmts an error, etc.

What can we do now to hide this sidebar if the theme does not have a `theme.json`?